lib/features/digital_guide_view/general_info/presentation/widgets/digital_guide_features_section.dart

```diff
@@ -0,0 +1,93 @@
+import "package:flutter/material.dart";
+import "package:flutter/widgets.dart";
+
+import "../../../../../config/ui_config.dart";
+import "../../../../../utils/context_extensions.dart";
+import "../../../../../widgets/my_expansion_tile.dart";
+import "../../../amenities/presentation/amenities_expansion_tile_content.dart";
+import "../../../localization/presentation/localization_expansion_tile_content.dart";
+import "../../../surrounding/presentation/surroundings_expansion_tile_content.dart";
+import "../../data/models/digital_guide_response.dart";
+
+class DigitalGuideFeaturesSection extends StatelessWidget {
+  const DigitalGuideFeaturesSection({required this.digitalGuideResponse});
+
+  final DigitalGuideResponse digitalGuideResponse;
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.only(left: 24, right: 24, top: 24, bottom: 8),
+      // TODO(Bartosh): don't use Column, use other Widget for better performance 
+      child: Column(
+        children: [
+          MyExpansionTile(
+            title: context.localize.localization,
+            children: [LocalizationExpansionTileContent()],
+          ),
+          const SizedBox(height: DigitalGuideConfig.heightMedium),
+          MyExpansionTile(
+            title: context.localize.amenities,
+            children: [
+              AmenitiesExpansionTileContent(
+                digitalGuideResponse: digitalGuideResponse,
+              ),
+            ],
+          ),
+          const SizedBox(height: DigitalGuideConfig.heightMedium),
+          MyExpansionTile(
+            title: context.localize.surroundings,
+            children: [
+              SurroundingsExpansionTileContent(
+                digitalGuideResponse: digitalGuideResponse,
+              )
+            ],
+          ),
+          const SizedBox(height: DigitalGuideConfig.heightMedium),
+          MyExpansionTile(
+            title: context.localize.transport,
+            children: [LocalizationExpansionTileContent()],
+          ),
+          const SizedBox(height: DigitalGuideConfig.heightMedium),
+          MyExpansionTile(
+            title: context.localize.entrances,
+            children: [LocalizationExpansionTileContent()],
+          ),
+          const SizedBox(height: DigitalGuideConfig.heightMedium),
+          MyExpansionTile(
+            title: context.localize.elevators,
+            children: [LocalizationExpansionTileContent()],
+          ),
+          const SizedBox(height: DigitalGuideConfig.heightMedium),
+          // there is no data provided in any endpoint for this expansion tile (toilets)
+          // This tile is not implemented on digital guide website XDDDD
+          // Can I skip it?
```

Comment:
Did you check other buildings?

Reply:
Sure, I don't see any endpoint or any JSON property for it in API. It's also empty on their website.

Reply:
I'll check that later. For now just leave it here